unidist/core/backends/mpi/core/common.py

```diff
@@ -258,50 +258,54 @@ def get_op_name(op):
     return op_name
 
 
-class MasterDataID(DataID):
+class MpiDataID(DataID):
     """
     Class for tracking data IDs of the main process.
 
-    Class extends ``unidist.core.backends.common.data_id.DataID`` functionality with a garbage collection.
+    The class extends ``unidist.core.backends.common.data_id.DataID`` functionality,
+    ensuring the uniqueness of the object to work correctly with MPI communication.
```

Comment:
I have no idea how to explain it briefly. This is not so simple, because to understand it, you need knowledge about local data storage and garbage collection between MPI processes.

Reply:
Left a suggestion.